package.json

```diff
@@ -39,6 +39,7 @@
     "toggle-switch-react-native": "^2.1.0"
   },
   "devDependencies": {
+    "babel-plugin-root-import": "^6.4.1",
```

Comment:
What is this for?

Reply:
It's so we can use ~ instead ../../ for imports but now that i think about it i didn't end up using anywhere